modules/creating-an-infra-node.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/cluster-tasks.adoc
+
+[id="creating-an-infra-node_{context}"]
+= Creating an infra node
+
+[IMPORTANT]
+====
+This procedure applies to user-provisioned infrastructure environments only.
+
+See Creating infrastructure machine sets for installer-provisioned infrastructure
```

Comment:
Shouldn't hard wrap, but I'm guessing that this was opened before we added those explicit guidelines? So no worries if so :)